mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -503,7 +503,11 @@ public void showProgressbar(boolean show) {
     @Override
     public void showClientInformation(final Client client) {
         if (client != null) {
-            setToolbarTitle(getString(R.string.client) + " - " + client.getLastname());
+            String lastName = client.getLastname();
+            if (lastName == null) {
+                lastName = client.getDisplayName();
+            }
+            setToolbarTitle(getString(R.string.client) + " - " + lastName);
             isClientActive = client.isActive();
```

Comment:
This client string requires two arg, it has two string placeholders. Don't you forget about them? Also does it possible to put string arg placeholder for las name as well and supply via getString() call?

Reply:
It is just for showing the display name of the client